pkg/auth/authorizer/abac/abac_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apiserver/pkg/authentication/user"
 	"k8s.io/apiserver/pkg/authorization/authorizer"
+	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
```

Comment:
Undo all new imports of this (applies to multiple files) and use `context.TODO()` in places where you do not have a context.

Reply:
done